code/game/objects/items/trader.dm

```diff
@@ -636,7 +636,7 @@
 										"Try again when you can relate to the intoxicating taste of blood.",
 										"What is a man? A miserable little pile of soft drinks.",
 										"You mortals all look underage to me. Pray tell, can you even manage a bottle?",
-										"Bah. You mock Le Confr�rie des Chevaliers du Tastevin with your plebian visage.",
+										"Bah. You mock Le Confr�rie des Chevaliers du Tastevin with your plebian visage.",
```

Comment:
RIP encoding.

Reply:
fug

Reply:
I'll try to fix that and the mapping errors tonight, away from my laptop for the next six hours or so.